test/test_subset.py

```diff
@@ -104,19 +104,49 @@ def test_grid_bounding_box_subset():
                 bbox_antimeridian[0], bbox_antimeridian[1], element=element)
 
 
-
-
 def test_uxda_isel():
     uxds = ux.open_dataset(GRID_PATHS[0], DATA_PATHS[0])
 
     sub = uxds['bottomDepth'].isel(n_face=[1, 2, 3])
 
     assert len(sub) == 3
 
+
 def test_uxda_isel_with_coords():
     uxds = ux.open_dataset(GRID_PATHS[0], DATA_PATHS[0])
     uxds = uxds.assign_coords({"lon_face": uxds.uxgrid.face_lon})
     sub = uxds['bottomDepth'].isel(n_face=[1, 2, 3])
 
     assert "lon_face" in sub.coords
     assert len(sub.coords['lon_face']) == 3
+
+
+def test_inverse_indices():
```

Comment:
One small suggestion
- When testing the `isel` functionality, please check that the stored inverse indices for the faces is the same one that is expecte.d